### PR TITLE
Validations model specs

### PR DESCRIPTION
**Pull request by Theophileaseh**
## In this project milestone, I implemented the following project requirements:

**Add the following validations:**

1. For the User model:

- Name must not be blank.
- PostsCounter must be an integer greater than or equal to zero.

2. For the Post model:

- Title must not be blank.
- Title must not exceed 250 characters.
- CommentsCounter must be an integer greater than or equal to zero.
- LikesCounter must be an integer greater than or equal to zero.

**Add unit specs for all of your models' methods and validations.**

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -1,3 +1,15 @@
 class Post < ApplicationRecord
   belongs_to :user
+  has_many :likes
+  has_many :comments
+
+  def most_recent_five_comments
+    comments.includes(:user).order(created_at: :desc).limit(5)
+  end
```
Comment (codepantha) [inline app/models/post.rb]: - [ ] Please add the following validations to your `Post` model as per requirements 👇 
- Title must not be blank.
- Title must not exceed 250 characters.
- CommentsCounter must be an integer greater than or equal to zero.
- LikesCounter must be an integer greater than or equal to zero.

File: spec/models_test/like_spec.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require '../rails_helper'
+
+RSpec.describe Like, type: :model do
+  subject { Like.new }
```
Comment (codepantha) [inline spec/models_test/like_spec.rb]: - [ ] I encounter some issues when trying to run your tests. It has to do with the way you're requiring '../rails_helper'. See error below 👇 Please get rid of the '../' to fix this issue.
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️ 

## Hi @Theophileaseh 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ No linter errors
✔️ Correct GitFlow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sinansevgi — APPROVED** [review]
Hi @Theophileaseh , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
All requested tests are covered and rspec works well 👍 
Test


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Happy Hacking! :computer: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
Sinan Sevgi

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**